README.md

```diff
@@ -491,6 +492,24 @@ and set:
 ```
 Replace `/home/kkaragiorgis/Projects/promil` to wherever you cloned `actions` repository.
 
+#### QA-Lint
+
+Example usage:
+```yaml
+...
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Run Linter
+        uses: PiwikPRO/actions/python/qa-lint@master
+        with:
+            python-version: "3.13.*"
+            poetry-version: "1.8.2"
+            ruff-version: "0.8.*"
```

Comment:
I would remove from example fixed versions to recommend follow default values. It will be easier to update later on too.